doc/tutorials/running_deb_as_serv_vm.rst

```diff
@@ -44,88 +44,93 @@ Validated Versions
 ******************
 
 - **Debian version:** 10.0 (buster)
```

Comment:
Can we update this? We mention Debian 10.0 but talk about 10.1 above in the doc and actually the download URL from the Debian download website only contains the 10.6 image. If that one (`Debian 10.6`) has been tested and validated, I'd recommend we use it throughout this doc.

Reply:
update to 10.1.